app/matplotlib.ipynb

```diff
@@ -233,22 +231,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, (ax1, ax2) = plt.subplots(1, 2)\n",
+    "\n",
+    "ax1.plot(x, y1, color=\"blue\")\n",
+    "ax1.set_title(\"sin X\")\n",
+    "ax1.set_xlabel(\"X value\")\n",
+    "ax1.set_ylabel(\"Sin x\")\n",
+    "\n",
+    "ax2.plot(x, y2, color=\"red\")\n",
+    "ax2.set_title(\"cos X\")\n",
+    "ax2.set_xlabel(\"X value\")\n",
+    "ax2.set_ylabel(\"Cos X\")"
```

Comment:
The figure title is missing. You should set a title for the entire figure using `fig.suptitle()` to meet the task requirements.